tooling/generate-registry-files.js

```diff
@@ -0,0 +1,93 @@
+const fs = require('fs');
+const mkdirp = require('mkdirp');
+const path = require('path');
+const didMethodRegex = /<tr>\n.*<td>\n\s+did:(.*):\n.*<\/td>\n.*<td>\s+(.*)\n.*<\/td>\n.*<td>\s+(.*)\n.*<\/td>\n.*<td>\n\s+(.*)\n.*<\/td>\n.*<td>\n\s+(.*)\n.*/g;
+const didSpecRegex = /<a href="(.*)">.*<\/a>/;
+const websiteRegex = /<a href="(.*)">.*<\/a>/;
+const emailRegex = /<a href="mailto:(.*)">.*<\/a>/;
+// set directories and paths
+const registryFile = path.join(__dirname, '../index.html');
+const methodsDir = path.join(__dirname, '../methods');
+const sectionDir = path.join(__dirname, '../sections');
+
+// create directories
+mkdirp.sync(methodsDir);
+mkdirp.sync(sectionDir);
+
+const validateRegistryEntry = require('./validation');
+
+// we migth consider not allowing html here
+const getUnsafeContact = (contact)=>{
```

Comment:
I feel that given the current poor showing for contact, and its ambiguous types (urls, company names, people names, emails)... we might consider removing it...

Reply:
I agree there's been a poor showing, but still think that it is good information to try to have